src/plugins/blocksmith/utils/convertToCanvasBlocks.js

```diff
@@ -37,7 +37,10 @@ export default (editorState, allowEmptyTextBlocks = false) => {
     blockRenderers: {
       [blockTypes.ATOMIC]: (block) => `${CANVAS_BLOCK_TOKEN}${JSON.stringify(block.getData().get('canvasBlock'))}`,
       [blockTypes.UNSTYLED]: (block) => {
-        if (allowEmptyTextBlocks && block.getText() === '') {
+        if (
+          allowEmptyTextBlocks
```

Comment:
@efjacobson maybe trim the block.getText() and look for empty string intead so you don't need regex at all